### PR TITLE
libmikmod: add homepage, change source, fix build

### PR DESCRIPTION
**Pull request by skmd0**
**Summary**

- Added homepage to package.yml
- Replaced mirror with direct source
- Fixed failing build

**Test Plan**

- Checked if homepage was added. 
- Downloaded from new source. 
- Built package successfully.

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/l/libmikmod/package.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 name       : libmikmod
 version    : 3.3.11.1
-release    : 11
+release    : 12
 source     :
-    - https://netix.dl.sourceforge.net/project/mikmod/libmikmod/3.3.11.1/libmikmod-3.3.11.1.tar.gz : ad9d64dfc8f83684876419ea7cd4ff4a41d8bcd8c23ef37ecb3a200a16b46d19
+    - https://sourceforge.net/projects/mikmod/files/libmikmod/3.3.11.1/libmikmod-3.3.11.1.tar.gz : ad9d64dfc8f83684876419ea7cd4ff4a41d8bcd8c23ef37ecb3a200a16b46d19
+homepage   : https://mikmod.sourceforge.net
 license    :
     - LGPL-2.0
```
Comment (algent-al) [inline packages/l/libmikmod/package.yml:8]: Please fix these licenses to correct format.

**Review by algent-al — CHANGES_REQUESTED** [review]

File: packages/l/libmikmod/package.yml (hunk 1 above)
Comment (skmd0) [reply packages/l/libmikmod/package.yml:8]: Hmm, their website links to LGPL-3, should I change those two licenses to that?

Comment (skmd0) [reply packages/l/libmikmod/package.yml:8]: Ah nevermind, the tarball source has 2 and 2.1. Will fix!

**Review by algent-al — APPROVED** [review]
LGTM, thanks!

**Review by algent-al — APPROVED** [review]
LGTM, thanks!

**Pull request merged**